tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -120,27 +120,50 @@ Feature: add users to group
       | Alice    | var/../etc       |
 
 
-  Scenario: normal user tries to add himself to a group
-    Given group "groupA" has been created
+  Scenario Outline: user other than the admin tries to add himself to a group
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And group "groupA" has been created
     When user "Alice" tries to add himself to group "groupA" using the Graph API
     Then the HTTP status code should be "401"
     And the last response should be an unauthorized response
+    Examples:
+      | role        |
+      | Space Admin |
+      | User        |
+      | Guest       |
 
 
-  Scenario: normal user tries to other user to a group
+  Scenario Outline: user other than the admin tries to add other user to a group
     Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has given "Brian" the role "<role>" using the settings api
     And group "groupA" has been created
     When user "Alice" tries to add user "Brian" to group "groupA" using the Graph API
     Then the HTTP status code should be "401"
     And the last response should be an unauthorized response
+    Examples:
+      | role        |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+
+  Scenario: admin tries to add user to a nonexistent group
+    When the administrator tries to add user "Alice" to a nonexistent group using the Graph API
+    Then the HTTP status code should be "404"
 
 
-  Scenario: admin tries to add user to a non-existing group
+  Scenario Outline: other than the admin tries to add user to a nonexistent group
```

Comment:
in line 157 admin tries to add the user so this might not be the accurate title?
```suggestion
  Scenario Outline: admin tries to add user to a nonexistent group
```

Reply:
`non-admin user tries to add user to a nonexistent group`

Reply:
non-admin can't be used because we have space admin and @SwikritiT sajan is right its about user other than admin

Reply:
@amrita-shrestha what I meant was the secnario is 
```feature
 Scenario Outline: user other than the admin tries to add user to a nonexistent group
    Given the administrator has given "Alice" the role "<role>" using the settings api
    When the administrator tries to add user "Alice" to a nonexistent group using the Graph API
    Then the HTTP status code should be "404"
    Examples:
      | role        |
      | Space Admin |
      | User        |
      | Guest       |
```
The title is `user other than the admin tries to add user to a nonexistent group` but the when step is
```feature
  When the administrator tries to add user "Alice" to a nonexistent group using the Graph API
```
So they don't match, should the when step be
```feature
When "Alice" tries to add user "Brian" to a nonexistent group using the Graph API
```